### ISSUE TITLE
The model of contrast

### ISSUE THREAD

**Issue by zerodohero** [vqdang/hover_net#53]
This work is amazing, I have a little question, do the other models you compare also use imagNet for pre-training?

**Comment by vqdang** [comment]
Hi, thank you for your interest in our work.

To answer your question, not all of them. We used ImageNet pretrained as much as we could where its applicable such as for Mask RCNN and all HoVerNet variants in the comparison. We didn't use it for other models because there was no pretrained weight available for them, such as MicroNet which was proposed by another segmentation paper, or the U-Net and its variants.

**Comment by zerodohero** [comment]
> Hi, thank you for your interest in our work.
> 
> To answer your question, not all of them. We used ImageNet pretrained as much as we could where its applicable such as for Mask RCNN and all HoVerNet variants in the comparison. We didn't use it for other models because there was no pretrained weight available for them, such as MicroNet which was proposed by another segmentation paper, or the U-Net and its variants.

Thank you for your reply. Have you ever tried to do pre-training without using ImagNet?How much of an improvement on this

**Comment by vqdang** [comment]
We haven't tried that. You can investigate it :)

**Comment by zerodohero** [comment]
thank you
